index.js

```diff
@@ -250,9 +249,7 @@ if ((os.platform() === 'win32' && require('is-administrator')()) || process.getu
                 }
             }
 
-            distro((error, dist) => {
-                return cb(null, error ? {} : dist);
-            });
+            return getDistro(cb);
```

Comment:
according to the function declaration, this expects 2 input params, `path` and `callback`, but only one is provided
upon inspection, it looks like there is some inherent contextual check around `os.platform()` so technically this will work, but this is very error-prone -- in fact based on what I can see right now, the contextual conditional check and where this code is being run seems to be in exact contradiction, which likely will just break your code

I will leave my comments around the specifics in the function declaration